model/src/main/kotlin/util/FindingsMatcher.kt

```diff
@@ -53,9 +53,7 @@ class FindingsMatcher(
     private fun getRootLicense(licenseFindings: Collection<LicenseFinding>): String =
         // TODO: This function should return a list of all licenses found in all license files instead of only a single
         // license.
-        licenseFindings.singleOrNull { finding ->
-            licenseFileMatcher.matches(finding.location.path)
-        }?.license.orEmpty()
+        licenseFindings.find { licenseFileMatcher.matches(it.location.path) }?.license.orEmpty()
```

Comment:
Wouldn't the correct behavior be to match the copyright with all root licenses and therefore change this function to return a list of root licenses? Or is this an intermediate step?

Reply:
It is an intermediate step.